portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro

```diff
@@ -406,7 +406,12 @@ definition {
 	macro openSiteTemplateInEditMode {
 		var templateID = SiteTemplates.getSiteTemplateID(siteTemplateName = "${siteTemplateName}");
 
-		Navigator.openWithAppendToBaseURL(urlAppend = "group/${templateID}?p_l_mode=edit");
+		if (isSet(oldestVersion)) {
```

Comment:
Suggest to replace oldestVersion with widgetPage, it's more readable and straightforward.

Reply:
I looked into the test and agree with you, **widgetPage** is clearer, I pushed the changes!